aggie-coding-club/Rev-Registration
# Redirect on null term

## Description

This makes the SchedulingPage redirect itself to the homepage if term is null (when someone enters ./schedule url).

## How to test

Visit ./schedule in incognito mode and it should open the homepage. Tested in automatic tests.

## Screenshots

Normal operation:
![normal](https://user-images.githubusercontent.com/49422137/96940197-517c2700-148c-11eb-9968-1f87804e5f8e.gif)

Redirect (go straight to schedules without cookies):
![redirect](https://user-images.githubusercontent.com/49422137/96940211-593bcb80-148c-11eb-8d54-4d7bfa7d0e2a.gif)

Refresh:
![refresh](https://user-images.githubusercontent.com/49422137/96940220-635dca00-148c-11eb-97ad-ec339ecd84b5.gif)

## Update:

Changes reverted, there is now slight flashing on redirect but no flashing on refresh. See above screenshots.

Closes #301
